templates/config.edn

```diff
@@ -356,14 +364,18 @@
 
  ;; Configure the Enter key behavior for
  ;; context-aware editing with DWIM (Do What I Mean).
- ;; :dwim/settings {
- ;;   :admonition&src?  true
- ;;   :markup?          false
- ;;   :block-ref?       true
- ;;   :page-ref?        true
- ;;   :properties?      true
- ;;   :list?            true
- ;; }
+;; context-aware Enter key behavior implies that pressing Enter will
+;; have different outcomes based on the context.
+;; For instance, pressing Enter within a list generates a new list item,
+;; whereas pressing Enter in a block reference opens the referenced block.
+ :dwim/settings
+{:admonition&src?  true  ;; default: true
+ :markup?          false ;; default: false
+ :block-ref?       true  ;; default: true
+ :page-ref?        true  ;; default: true
+ :properties?      true  ;; default: true
+ :list?            true  ;; default: true
+ }
```

Comment:
```suggestion
 ;; context-aware Enter key behavior implies that pressing Enter will
 ;; have different outcomes based on the context.
 ;; For instance, pressing Enter within a list generates a new list item,
 ;; whereas pressing Enter in a block reference opens the referenced block.
 :dwim/settings
{:admonition&src?  true  ;; default: true
 :markup?          false ;; default: false
 :block-ref?       true  ;; default: true
 :page-ref?        true  ;; default: true
 :properties?      true  ;; default: true
 :list?            true  ;; default: true
 }
```

Reply:
Also I still think each of these could use a quick comment describing what each one does. I have no idea what "admonition&src" is, for example.

Reply:
And it's not clear exactly in what context each of these is talking about